clang/include/clang/Sema/Sema.h

```diff
@@ -997,6 +987,11 @@ class Sema final : public SemaBase {
     return *OpenACCPtr;
   }
 
+  SemaOpenMP &OpenMP() {
```

Comment:
```suggestion
  SemaOpenMP &getOpenMP() {
```

Reply:
This was discussed back when we started splitting `Sema` up: https://github.com/llvm/llvm-project/pull/84184#discussion_r1520027821. We can reopen the discussion if you insist, otherwise I'd like to keep it consistent with CUDA, HLSL, etc.